src/config/api.js

```diff
@@ -37,4 +38,14 @@ export const apiSetup = dispatch => {
   });
 };
 
+api.addMonitor(response => {
+  const camelCaseSerializer = new CamelcaseSerializer();
+  response.data = camelCaseSerializer.serialize(response.data);
```

Comment:
what if the response doesnt have data? you need to validate it because this is GLOBAL, and applies to ALL requests